src/language_features/document_symbol.rs

```diff
@@ -236,8 +236,7 @@ fn editor_next_or_prev_for_details(
         work_done_progress_params: Default::default(),
     };
 
-    // This context is shown at the top of the modal.
-    let context = format!(
+    let modal_heading = format!(
         "line {}:{}:{{+b@KindAndName}}{:?} {}{{KindAndName}} \
```

Comment:
BTW instead of KindAndName we could use one of our established faces like `InfoHeader`